src/betterproto/plugin/compiler.py

```diff
@@ -33,5 +33,5 @@ def outputfile_compiler(output_file: OutputTemplate) -> str:
 
     return black.format_str(
         template.render(output_file=output_file),
-        mode=black.FileMode(target_versions={black.TargetVersion.PY37}),
+        mode=black.FileMode(),
```

Comment:
```suggestion
        mode=black.Mode(),
```
Might be a good idea to update for this https://github.com/psf/black/blob/main/src/black/__init__.py#L98-L99